package.json

```diff
@@ -26,7 +26,8 @@
     "files": [
         "dist/",
         "index.d.ts",
-        "index.js"
+        "index.js",
+        "static/"
```

Comment:
Is this really the only way to include a screenshot in a package? Does this mean every user has to download the screenshot as well when they install it?

Also, please update the company and email in the author field :)

Reply:
Apparently there is: https://stackoverflow.com/questions/69381117/publishing-images-to-be-included-in-readme-md-on-npm. You just have to add some additional properties to the package.json. I can not test it tho. I've changed it according to the stack overflow post and pushed it